### ISSUE TITLE
Create filter when clicking on property in the traces

### ISSUE THREAD

**Issue by skull8888888** [lmnr-ai/lmnr#192]
<img width="1445" alt="Screenshot 2024-11-09 at 10 42 14 PM" src="https://github.com/user-attachments/assets/599794c0-b8bd-4282-ab84-da294af1d1e4">

For instance, in the screenshot above, I want to be able to click on "capital_finder" top span name, and then it would create filter on top span name with value equals to the "capital_finder"

**Comment by dinmukhamedm** [comment]
Yes, @sarthakvijayvargiya this is correct. We'd probably like something similar for top level span (type) too. The filter is currently broken though, I will fix it ASAP

**Comment by dinmukhamedm** [comment]
@sarthakvijayvargiya sorry I forgot to update you that we fixed the filter. Any updates from your side?

**Comment by sarthakvijayvargiya** [comment]
@dinmukhamedm I'm currently working on  issue #79 and won't be able to focus on this one for now. Please unassign me from this issue temporarily. If it's still open once I’m done, I’ll pick it up again.

**Comment by nidhi-wa** [comment]
@dinmukhamedm I’ve implemented the Auto Filter feature in this PR. I’d like to hear your thoughts and suggestions.#317
